tad/models/task.py

```diff
@@ -0,0 +1,30 @@
+from pydantic import BaseModel, ValidationInfo, field_validator
+from pydantic import Field as PydanticField  # type: ignore
+from sqlmodel import Field as SQLField  # type: ignore
+from sqlmodel import SQLModel
+
+
+class Task(SQLModel, table=True):
+    id: int = SQLField(default=None, primary_key=True)
+    title: str
+    description: str
+    sort_order: float
+    status_id: int | None = SQLField(default=None, foreign_key="status.id")
```

Comment:
i would make the status_id a status with an enum

Reply:
I am not sure how dynamic status will be. Maybe it is not needed to be a table after all and and enum will be sufficient. For now I'd like to keep it this way.

Reply:
Ok!